arch/arm/configs/bcm2835_defconfig

```diff
@@ -2,13 +2,13 @@
 CONFIG_SYSVIPC=y
 CONFIG_NO_HZ=y
 CONFIG_HIGH_RES_TIMERS=y
+CONFIG_PREEMPT_VOLUNTARY=y
```

Comment:
Yes it's for testing convenience. I disabled the C driver and enabled the Rust driver for the Pi's hwrng. Otherwise both drivers could "fight" for a devicetree `compatible` string. Not even sure what would happen in such a case.

Reply:
It may make sense to fork this file into say `bcm2835_rust_defconfig`.